user-manual/transfer/clean-up-names.rst

```diff
@@ -0,0 +1,91 @@
+.. _clean-up-names:
+
+==============
+Clean up names
+==============
+
+The clean up names microservice runs twice during processing - once on the
+Transfer tab and once on the Ingest tab. The purpose of this microservice is to
+ensure that Archivematica's tools and processes do not fail because of
+characters that appear in file or directory names.
+
+Archivematica groups a wide variety of tools together to create preservation
+workflows. Some of the tools that Archivematica uses have a narrowly-defined
+scope in terms of what the tool considers to be a valid character for a file or
+directory name. Encountering a character that the tool considers to be invalid
+can cause the tool to fail, which can halt processing and interfere with normal
+operation of Archivematica.
+
+To prevent these kinds of tool failures from happening, Archivematica implements
+a script that changes file and directory names to conform to requirements of the
+most restrictive tools. Valid characters are defined by `specific code`_ in
+Archivematica:
+
+``-_.()abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789``
+
+The script tries to replace any character that is not included in the above list
+with its nearest equivalent - for example, the character ``Č`` is replaced by
+``C``. Where the script cannot find a close equivalent, the character is
+replaced by an underscore. The change is documented in the METS file.
+
+.. csv-table::
+   :file: _csv/file-name-edits.csv
+   :header-rows: 1
```

Comment:
This is a very cool way to include a table!

Reply:
@currmie taught us this! :tada: